server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java

```diff
@@ -21,7 +21,8 @@
 
 import org.sonar.api.platform.Server;
 import org.sonar.api.platform.ServerStartHandler;
-import org.sonar.api.server.ServerSide;
+import org.sonar.api.ce.ComputeEngineSide;
+import org.sonar.ce.monitoring.CEQueueStatus;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.ce.monitoring.CEQueueStatus'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)